META.json

```diff
@@ -74,15 +75,27 @@
    "provides" : {
       "NewFangle::Agent" : {
          "file" : "lib/NewFangle/Agent.pm",
-         "version" : "0.005"
+         "version" : "0.006"
       },
       "NewFangle::Agent::Config" : {
          "file" : "lib/NewFangle/Agent/Config.pm",
-         "version" : "0.005"
+         "version" : "0.006"
+      },
+      "NewFangle::Agent::SegmentStarter::DBI::st" : {
```

Comment:
We definitely need these, but maybe we can rid of the other changes, caused by the different dzil version?

Reply:
Got the latest `dzil` from CPAN which fixes this.